### ISSUE TITLE
Better repository structure

### ISSUE THREAD

**Issue by dyorgio** [neuroph/neuroph#51]
Please,

Separate IDE and library sources into two repositories.

It will help to download source of library.

**Comment by sevarac** [comment]
I agree. Just want to put together some contributions that are ongoing and after that we'll do it
Thanks
